themes/bootstrap3/templates/Recommend/SideFacets/facet.phtml

```diff
@@ -0,0 +1,55 @@
+<?php
+  $this->results = $results = $this->recommend->getResults();
```

Comment:
There are a lot of variables being defined here, and some could easily be passed in from SideFacets.phtml instead. Are they redefined for compatibility with the SideFacetsDeferred AJAX handler? (I imagine you've probably already thought this through, but just double-checking if anything can be simplified here to reduce duplicate work).

Reply:
In normal facets these are defined in SideFacets.phtml, but deferred retrieval renders facet.phtml directly. I tried to keep the changes to minimum for back-compatibility (e.g. if someone has customized the sub-templates previously called directly from SideFacets.phtml), but some cleanup could be done at least for version 6.

Reply:
Thanks for the clarification. That's what I suspected, but just wanted to double-check.